kbase-extension/static/kbase/js/common/cellComponents/fieldTableCellWidget.js

```diff
@@ -192,20 +192,46 @@ define([
             }
         }
 
+        /**
+         *
+         * @param {object} rows
+         */
         function setDuplicateValue(rows) {
             state.isDuplicate = true;
             let message = 'duplicate value';
             let rowMessage = '';
+            let tabMessage = '';
+            let tabPrefix = ' on tab';
+            let rowTabSeparator = '';
+            let plural = false;
 
-            if (rows) {
+            if (rows && rows.thisTab && rows.thisTab.length) {
                 rowMessage = ' on row';
-                if (rows.length > 1) {
-                    message += 's';
+                if (rows.thisTab.length > 1) {
                     rowMessage += 's';
+                    plural = true;
+                }
+                rowMessage += ' ' + StringUtil.arrayToEnglish(rows.thisTab);
+            }
+
+            if (rows && rows.otherTabs) {
+                const tabs = Object.keys(rows.otherTabs);
+                // don't use rows for now, as there's something funny about the reporting
```

Comment:
were you able to reproduce the strangeness in a test?

Reply:
No, but I think it stems from how and when validation messages get sent, or not. And if they get sent with some invalid content, they shouldn't get stored in the data model. Some seem to seep through, though.

I'll put a TODO in to try to track it down, but I think that's going to be a deep, dark, rabbit hole of bus message mayhem.

Reply:
Fixed it! Your comment on the configure tab was the bug. 🎉